apps/web/src/actions/register.ts

```diff
@@ -7,16 +7,11 @@ import { z } from "zod";
 
 import { auth } from "@echo-webkom/auth";
 import { db } from "@echo-webkom/db";
-import {
-  answers,
-  registrations,
-  users,
-  type AnswerInsert,
-  type SpotRange,
-} from "@echo-webkom/db/schemas";
+import { answers, registrations, users, type AnswerInsert } from "@echo-webkom/db/schemas";
 
 import { doesArrayIntersect } from "@/lib/array";
 import { registrationFormSchema } from "@/lib/schemas/registration";
+import { getUserSpotRange } from "@/utils/getUserSpotRange";
```

Comment:
Vi bruker kebab-case for filnavn.

Og har du glemt å legge til denne filen i PR?